packages/kbn-plugin-generator/sao_template/template/index.js

```diff
@@ -9,7 +9,10 @@ export default function (kibana) {
       app: {
         title: '<%= startCase(name) %>',
         description: '<%= description %>',
-        main: 'plugins/<%= kebabCase(name) %>/app'
+        main: 'plugins/<%= kebabCase(name) %>/app',
+        <% if (generateScss) { %>
+        styleSheetPath: `${__dirname}/public/app.scss`,
```

Comment:
This isn't going to work on windows, it needs to have the right path separators all the way through.

```js
require('path').resolve(__dirname, 'public/app.scss')
```